.github/ISSUE_TEMPLATE/issue report.yml

```diff
@@ -4,52 +4,109 @@ title: "[Issue]: "
 labels: [bug, triage]
 body:
   - type: markdown
+    id: introduction
     attributes:
       value: |
-        Thanks for taking the time to report an issue. Before submitting a report, please do the following:
-        1. Please head to our forum or chat rooms and troubleshoot with volunteers if you haven't already. Links can be found here: https://jellyfin.org/contact/
-        2. Please search the bug tracker for similar issues. If you do find one, please comment there instead of opening a new bug report.
-        3. If you decide to open a new report, please provide as much detail as possible.
-        4. Please **ONLY** report **ONE** issue per report. If you are experiencing multiple issues, please open multiple reports.
+        ### Thank you for taking the time to report an issue!
+        Please keep in mind that Jellyfin is a [free and open-source](https://jellyfin.org/docs/general/about) project, made up entirely and exclusively of **volunteers** who donate their free time to the project.
+  - type: checkboxes
+    id: before-posting
+    attributes:
+      label: "This issue respects the following points:"
+      description: All conditions are **required**. Failure to comply with any of these conditions may cause your issue to be closed without comment.
+      options:
+        - label: This is a **bug**, not a question or a configuration issue. We recommend visiting our forum or chat rooms and troubleshoot with volunteers first. The links can be found [here](https://jellyfin.org/contact/).
```

Comment:
```suggestion
        - label: This is a **bug**, not a question or a configuration issue. You should visit our forum or chat rooms and troubleshoot with volunteers first. The links can be found [here](https://jellyfin.org/contact/). Please don't try to outsmart the triage team. It is very obvious when someone didn't troubleshoot with volunteers before opening an issue.
```

Reply:
I do not like that passive aggressiveness of the latter part. I can change the "recommend" into something more forceful, however.

Reply:
I don't have any problem removing that latter part, but make the recommend more forceful.

Reply:
Yea none of that stuff at the end is needed. Not a tone we want. And "Please" instead of "you should".

My mention of force was only in regards to the wording above, i.e. saying "if you don't even bother filling out our templates, we're going to close your issue and not even comment on it".